.github/workflows/ci.yml

```diff
@@ -26,6 +26,17 @@ jobs:
             exclude_regress: 3-4
             run_regression_args: --tester base --tester model --tester synth --tester abduct --tester dump
 
+          - name: ubuntu:production-safe
```

Comment:
Should we use `ubuntu:safe-mode` instead, so that the part after the colon matches the name of the build?

Reply:
Yes